doc/source/api/metadata.rst

```diff
@@ -110,13 +110,13 @@ Durations
 =========
 
 
-Durations are the most common form of time intervals. They do not reference to
+Durations are the most common form of time intervals. They do not reference make to
```

Comment:
Think this was intense to read "... They do not make reference to ..." (though I think the original was o.k.?).

To emphasise the difference, the next sub-header could be changed to "(Anchored) Time Intervals" or such?